ohai.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency "ipaddress"
   s.add_dependency "wmi-lite", "~> 1.0"
   s.add_dependency "ffi", "~> 1.9"
+  s.add_dependency "chef-config", "~> 12.4.1"
```

Comment:
It's going to be annoying having to bump the version here all the time. Any objections to letting be '~> 12.4`?